javascript-typing-tutor/typingGame.js

```diff
@@ -0,0 +1,55 @@
+const allLetters = document.querySelectorAll('span');
+const spanSp = document.querySelectorAll('.space');
+
+// borders
+
+let currVal = 0;
+
+// allLetters[0].className = 'bl'
+
+window.addEventListener('keydown', (event) => {
+  const currKey = event.key;
+  const currKeyPoint = allLetters[currVal];
+  if (currKey === currKeyPoint.textContent) {
+    currKeyPoint.className = 'no-more-border correct-key';
+    console.log(currKeyPoint.nextElementSibling);
```

Comment:
Remove all `console.log`s before submitting your code. They are only for testing / debugging purposes